draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -36,6 +36,14 @@ author:
     city: Cupertino, California 95014
     country: United States of America
     email: cawood@apple.com
+ -
+    ins: A. Faz-Hernandez
```

Comment:
We're keeping the list alphabetical, so please move your name up! :)

Reply:
Done